### DISCUSSION TITLE
Will SFSafariViewController be supported on iOS?

### DISCUSSION THREAD

**Discussion by JPGrefaldo** [wix/Detox#4530] [General]
Currently `WKWebview` i.e. iOS WebView is supported with latest detox version kudos!
Unfortunately there are apps that use in-app browser specially in iOS  i.e. `SFSafariViewController`.

**Comment by asafkorem** [comment]
Hey @JPGrefaldo, thanks 🙂 it will be. Can't provide an ETA at the moment, but it's already planned.
 
See: https://github.com/wix/Detox/issues/4465
